src/version.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-export const VERSION = '0.1.0';
+export const VERSION = '0.4.0';
```

Comment:
Can you add an automated check in CI to prevent version mismatch?

Reply:
I can create a ticket for it :)